src/main/java/org/isf/examination/manager/ExaminationBrowserManager.java

```diff
@@ -299,9 +299,9 @@ public String getDiuresisDescriptionKey(String description) {
 	 */
 	protected List<OHExceptionMessage> validateExamination(PatientExamination patex) {
 		List<OHExceptionMessage> errors = new ArrayList<>();
-		if (patex.getPex_note().length() > 300) {
+		if (patex.getPex_note().length() > PatientExamination.PEX_NOTE_LENGHT) {
```

Comment:
This variable is spelled wrong; `LENGHT` should be `LENGTH`.

Reply:
Thanks!